<launch>
  <group ns="/s4n1/device/sensor0/head_camera/">
    <node pkg="s4_operation" type="s4_operation_image_overlay" name="s4_operation_image_overlay" output="screen">
      <remap from="scan" to="/s4n1/device/sensor0/front_laser/scan_filtered"/>
      <remap from="voltage" to="/s4n1/device/body/battery/s4_body_battery/voltage"/>
      <remap from="aim_point" to="/s4n1/device/gun0/command"/>
      <remap from="objects" to="/s4n1/detection/objects"/>
    </node>
    <node name="image_view" pkg="image_view" type="image_view" >
      <remap from="image" to="image_overlay"/>
    </node>
  </group>
</launch>

